### DISCUSSION TITLE
第1节课笔记(21班)

### DISCUSSION THREAD

**Discussion by mattheliu** [InternLM/Tutorial#137] [笔记]
请提交笔记链接
Please submit the link to the notes.

**Comment by nanamma** [comment]
hi~

**Comment by JinProton** [comment]
https://zhuanlan.zhihu.com/p/676862495

**Comment by cqu-student** [comment]
想学习大语言模型(LLM)，应该从哪个开源模型开始？ - 空.白的回答 - 知乎
https://www.zhihu.com/question/608820310/answer/3355928501

**Comment by jayjayhust** [comment]
https://github.com/jayjayhust/InternLM-Tutorial/blob/main/Lesson1.md

**Comment by harryguantj** [comment]
课程笔记：
书生·浦语大模型实战营:Ch1-书生·浦语大模型全链路开源体系
https://blog.csdn.net/gyzlhg/article/details/135501212

**Comment by Fly-dream12** [comment]
https://www.zhihu.com/question/634064494/answer/3357481706

**Comment by HaroldChen19** [comment]
https://github.com/HaroldChen19/InternLM-Boot-Camp/blob/master/Lesson_1/Note.md

**Comment by enzo547** [comment]
https://www.bilibili.com/read/mobile?id=29440691&spm_id_from=333.999.0.0

**Comment by TonyTnT** [comment]
https://zhuanlan.zhihu.com/p/677359089

**Reply by mattheliu** [reply]
同学，请问你填写表单了吗

**Reply by TonyTnT** [reply]
忘记了 刚填了

**Comment by yunhuisong2023** [comment]
https://reflective-value-ac9.notion.site/1-b0a073490ef74c81ba4a5cd1362d882d

**Comment by XuandYu000** [comment]
https://www.cnblogs.com/viewoverlooking/p/17962560

**Comment by BoogalooYue** [comment]
**Lesson1 : 书生·浦语大模型全链路开源体系**
[笔记链接](https://www.yuque.com/g/boogalooyue/knowledge/nuxp3gm0w2a4mud7/collaborator/join?token=fmTDPQbYWbfROFi5&source=doc_collaborator#)

**Comment by dddxs** [comment]
hi~ Thanks！
https://github.com/dddxs/blog/issues/1#issue-2080428649

**Comment by water2silver** [comment]
https://zhuanlan.zhihu.com/p/677721249?

**Comment by KyleHuang9** [comment]
https://github.com/KyleHuang9/InternLM-tutorial/blob/main/Lesson1/node.md

**Comment by YongXie66** [comment]
https://github.com/YongXie66/InternLM_tutorial_copy/blob/main/notes/Lesson1.md

**Comment by mattheliu** [comment]
https://github.com/mattheliu/gitblog/issues/1
